pkg/cli/cli.go

```diff
@@ -82,7 +82,8 @@ const (
 const (
 	gcpMachineTypeFlag    = "gcp_machine_type"
 	gcpMinCPUPlatformFlag = "gcp_min_cpu_platform"
-	gcpBootDiskSizeGB     = "gcp_boot_disk_size_gb"
+	gcpBootDiskSizeGBFlag = "gcp_boot_disk_size_gb"
```

Comment:
Thanks for noticing `gcpBootDiskSizeGB` is missing the `Flag` suffix. However, let's have this fix in a different commit to keep this commit focused on accelerator config.